src/canmatrix/canmatrix.py

```diff
@@ -94,15 +106,18 @@ def normalizeValueTable(table):
 @attr.s(cmp=False)
 class Signal(object):
     """
-    contains on Signal of canmatrix-object
-    with following attributes:
-            name, startBit,size (in Bits)
-            is_little_endian (1: Intel, 0: Motorola)
-            is_signed ()
-            factor, offset, min, max
-            receiver  (Boarunit/ECU-Name)
-            attributes, _values, _unit, _comment
-            _multiplex ('Multiplexor' or Number of Multiplex)
+    Represents a Signal in CAN Matrix.
+
+    Signal has following attributes:
+
+    * name
+    * startBit, size (in Bits)
+    * is_little_endian (1: Intel, 0: Motorola)
+    * is_signed (bool)
+    * factor, offset, min, max
+    * receiver  (Boarunit/ECU-Name)
+    * attributes, _values, unit, comment
+    * _multiplex ('Multiplexor' or Number of Multiplex)
```

Comment:
Is this a standard doc form?  I can't say I've done much with docs but if we are filling this stuff in it might be worth finding doc generation that can work with attrs.

Reply:
Yes. Try ctrl+Q in pycharm, it gets rendered.

Reply:
I use pycharm, but it wouldn't be my primary target when picking a doc format.

Reply:
:-) No idea but I hope the sphinx can be converted to another format "automatically"